frontend/src/main/scala/akkaviz/frontend/components/LinkStateTab.scala

```diff
@@ -0,0 +1,103 @@
+package akkaviz.frontend.components
+
+import akkaviz.frontend.{Router, FrontendUtil, ActorLink, ActorPath}
+import akkaviz.frontend.ActorRepository.ActorState
+import akkaviz.{rest, protocol}
+import org.scalajs.dom
+import org.scalajs.dom.ext.Ajax
+import org.scalajs.dom.{Element => domElement, console}
+import rx.{Ctx, Rx, Var}
+
+import scala.concurrent.Future
+import scala.scalajs.js
+import scala.scalajs.js.{Date, JSON}
+import scalatags.JsDom.all._
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class LinkStateTab(link: ActorLink)(implicit co: Ctx.Owner) extends ClosableTab {
+
+  import LinkStateTab._
+  import akkaviz.frontend.PrettyJson._
+
+  import scalatags.rx.all._
+
+  val tabId = stateTabId(link)
+  val name = s"${FrontendUtil.shortActorName(link.from)} → ${FrontendUtil.shortActorName(link.to)}"
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Non ascii characters are not allowed](https://www.codacy.com/app/lustefaniak/akka-viz/file/2256576087/issues/source?bid=3121282&fileBranchId=3181785#l25)